### PR TITLE
Make pull request title formats more readable

### PR DESCRIPTION
**Pull request by yvanzo**
- Split the different cases off from paragraph,  as list items;
- Format the examples as code blocks to highlight these;
- Collapse the resulting list to avoid lengthening the overview.

### CODE REVIEW

File: GitHub.md [hunk 1]
```diff
@@ -11,12 +11,26 @@ We follow the "typical" GitHub workflow when contributing changes:
...
+   Make sure that the title of your pull request is descriptive.
+   <details>
+   <summary><i>Click to toggle formats for referencing ticket(s) in pull request’s title</i></summary>
+
+     * If you are fixing an issue that exists in our bug tracker, reference it like this:
+       ```
+       MBS-4635: Allow replacing images
+       ```
+       (Not `[MBS-4635] - Allow replacing images` or `Allow replacing images (MBS-4635)` or `MBS-4635`.)
+     * If you are fixing several issues at once, reference it like this:
```
Comment (reosarevok) [inline GitHub.md]:
```suggestion
     * If you are fixing several issues at once, reference them like this:
```

**Review by reosarevok — DISMISSED** [review]
Neat! I like the "click here for more" idea.

File: GitHub.md [hunk 2]
```diff
@@ -11,12 +11,26 @@ We follow the "typical" GitHub workflow when contributing changes:
...
+       ```
+       MBS-9477, MBS-10568, MBS-10569: DiscID tab improvements
+       ```
+     * In the rare case you are partially fixing an issue, reference it like this:
+       ```
+       MBS-10365 (III): Convert EventList to react-table
+       ```
+     Properly referenced tickets will be updated automatically.
```
Comment (MonkeyDo) [inline GitHub.md]:
```suggestion
     Properly referenced tickets will be automatically linked and updated in our bug tracker.
```

**Review by MonkeyDo — COMMENTED** [review]
Good improvement, thank you !

File: GitHub.md (hunk 1 above)
Comment (yvanzo) [reply GitHub.md]: Applied with commit https://github.com/metabrainz/guidelines/pull/23/commits/1abea135ddbf74278c1e6f277e9d74a2d6358db6

File: GitHub.md (hunk 2 above)
Comment (yvanzo) [reply GitHub.md]: Applied in commit https://github.com/metabrainz/guidelines/pull/23/commits/a6ce0f81919a41d12cb458a487486976d363dd5c

**Review by MonkeyDo — APPROVED** [review]
Ace! 🚀

**Pull request merged**